test/unittests/eof_validation_test.cpp

```diff
@@ -918,35 +897,31 @@ TEST_F(eof_validation, callf_into_nonreturning)
 {
     // function 0: (0, non-returning) : CALLF{1} STOP
     // function 2: (1, non-returning) : STOP
-    add_test_case(
-        "EF0001 010008 02000200040001 040000 00 00800000 00800000 "
-        "E3000100 00",
+    add_test_case(eof_bytecode(callf(1) + OP_STOP).code(OP_STOP, 0, 0x80, 0),
         EOFValidationError::callf_to_non_returning_function);
 }
 
 TEST_F(eof_validation, jumpf_equal_outputs)
 {
-    const auto code =
-        bytecode{"ef0001 01000c 020003 0004 0003 0004 040000 00 00800003 00030000 00030003"_hex} +
-        OP_CALLF + "0001" + OP_STOP + OP_JUMPF + "0002" + 3 * OP_PUSH0 + OP_RETF;
-
-    add_test_case(code, EOFValidationError::success);
+    add_test_case(eof_bytecode(callf(1) + OP_STOP, 3)
+                      .code(jumpf(2), 0, 3, 0)
+                      .code(3 * OP_PUSH0 + OP_RETF, 0, 3, 3),
+        EOFValidationError::success);
 }
 
 TEST_F(eof_validation, jumpf_compatible_outputs)
 {
-    const auto code =
-        bytecode{"ef0001 01000c 020003 0004 0005 0004 040000 00 00800005 00050002 00030003"_hex} +
-        OP_CALLF + "0001" + OP_STOP + 2 * OP_PUSH0 + OP_JUMPF + "0002" + 3 * OP_PUSH0 + OP_RETF;
-
-    add_test_case(code, EOFValidationError::success);
+    add_test_case(eof_bytecode(callf(1) + OP_STOP, 5)
+                      .code(2 * OP_PUSH0 + jumpf(2), 0, 5, 2)
+                      .code(3 * OP_PUSH0 + OP_RETF, 0, 3, 3),
+        EOFValidationError::success);
 }
 
 TEST_F(eof_validation, jumpf_incompatible_outputs)
 {
-    const auto code =
-        bytecode{"ef0001 01000c 020003 0004 0005 0004 040000 00 00800003 00030002 00050003"_hex} +
-        OP_CALLF + "0001" + OP_STOP + OP_JUMPF + "0002" + 5 * OP_PUSH0 + OP_RETF;
+    const auto code = eof_bytecode(callf(1) + OP_STOP, 3)
+                          .code(jumpf(2) + 2 * OP_PUSH0, 0, 3, 2)
+                          .code(3 * OP_PUSH0 + OP_RETF, 0, 5, 3);
```

Comment:
this looks wrong, and I think there was mistake in `max_stack_height` of 2nd section before
```suggestion
                          .code(jumpf(2), 0, 3, 0)
                          .code(5 * OP_PUSH0 + OP_RETF, 0, 5, 3);
```